### PR TITLE
Added gcis:findingStatement

### PR DESCRIPTION
**Pull request by justgo129**
Resolves https://github.com/USGCRP/gcis-ontology/issues/83

### CODE REVIEW

**Comment by zednis** [comment]
:+1:

**Comment by justgo129** [comment]
@rewolfe what do you think?

**Comment by rewolfe** [comment]
+1

On Fri, Jul 24, 2015 at 8:34 AM, justgo129 notifications@github.com wrote:

> @rewolfe https://github.com/rewolfe what do you think?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/USGCRP/gcis-ontology/pull/91#issuecomment-124505965.

## 

Robert Wolfe, NASA GSFC @ USGCRP, o: 202-419-3470, m: 301-257-6966

**Comment by justgo129** [comment]
Excellent.  @bduggan please merge.

File: gcis.ttl [hunk 1]
```diff
@@ -709,6 +709,12 @@ gcis:findingProcess a owl:DatatypeProperty ;
 	rdfs:comment "A finding has a description of the finding generation process." ;
 	rdfs:domain gcis:Finding ;
 	rdfs:range xsd:string .
+	
+gcis:findingStatement a owl:DatatypeProperty ;
+	rdfs:label "Key Finding" ;
+	rdfs:comment "The text of a Finding in a Highly Influential Scientific Document." ;
```
Comment (bduggan) [inline gcis.ttl]: "Finding" should not be capitalized, and I'm not sure I understand the HISD capitalization; is this the same as a HISA?

**Comment by bduggan** [comment]
On Friday, July 24, justgo129 wrote: 

> Excellent.  @bduggan please merge.

@justgo129, others can merge for the gcis-ontology repo.

**Comment by justgo129** [comment]
Good catches on both.

**Comment by justgo129** [comment]
Merged #91.

**Pull request merged**